cinema_service/settings.py

```diff
@@ -10,19 +10,18 @@
 https://docs.djangoproject.com/en/4.0/ref/settings/
 """
 
+from datetime import timedelta
 from pathlib import Path
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
-
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = ("django-insecure-6vubhk2$++agnctay_4pxy_8cq)"
+              "mosmn(*-#2b^v4cgsh-^!i3")
```

Comment:
The SECRET_KEY is hardcoded, which is not secure for production environments. Consider using environment variables or a secrets manager to handle sensitive information securely.